pkg/kwokctl/runtime/binary/cluster.go

```diff
@@ -501,37 +509,29 @@ func (c *Cluster) Down(ctx context.Context) error {
 	etcdPath := utils.PathJoin(bin, "etcd")
 	prometheusPath := utils.PathJoin(bin, "prometheus")
 
-	err = utils.ForkExecKill(ctx, conf.Workdir, kwokControllerPath)
-	if err != nil {
-		c.Logger().Printf("failed to kill kwok: %s", err)
-	}
-
-	err = utils.ForkExecKill(ctx, conf.Workdir, kubeSchedulerPath)
-	if err != nil {
-		c.Logger().Printf("failed to kill kube-scheduler: %s", err)
-	}
-
-	err = utils.ForkExecKill(ctx, conf.Workdir, kubeControllerManagerPath)
-	if err != nil {
-		c.Logger().Printf("failed to kill kube-controller-manager: %s", err)
-	}
-
-	err = utils.ForkExecKill(ctx, conf.Workdir, kubeApiserverPath)
-	if err != nil {
-		c.Logger().Printf("failed to kill kube-apiserver: %s", err)
+	componentPaths := map[string]string{
```

Comment:
```suggestion
	componentPaths := []string{
```

Let's use string slice here